.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: bash
+sudo: true
```

Comment:
If the needed packages are white listed, you can use the apt addon and still do the build in a container (which is much faster). See http://docs.travis-ci.com/user/migrating-from-legacy/#Adding-APT-Packages